examples/nas/dynamic/sc-filesystem.yaml

```diff
@@ -8,17 +8,19 @@ mountOptions:
 parameters:
   # volumeAs is optional, should be 'filesystem' or 'subpath', default value is 'subpath'
   volumeAs: filesystem
+  fileSystemType: standard
   # storageType is optional, should be 'Performance' or 'Capacity', default value is 'Performance'
-  #storageType: Performance
+  storageType: Performance
+  regionId: cn-hangzhou
   # zoneId is optional, default value is the kubernetes cluster's worker's zoneId
-  #zoneId: cn-hangzhou-f
+  zoneId: cn-hangzhou-f
```

Comment:
keep the zoneId and regionId commented, so that the reader can easily copy them into their cluster?